src/core/reducers/index.ts

```diff
@@ -9,15 +10,32 @@ import ui from './ui';
 
 export type Action = Actions.RefreshState;
 
-export const rootReducer = redux.combineReducers<Store.State>({
-  isRunning,
-  session,
-  data: undoable(data, {
-    limit: 5,
-    filter: ({ type }) => type === Actions.SAVE_STATE
-  }),
-  ui,
-});
+export const undoWithoutHistory = (store) => {
+  return (state, action) => {
+    const limit: number = ConfigAdapter.extractHistoryLength(store);
+    const config = {
+      limit,
+      filter: includeAction(Actions.SAVE_STATE),
+    };
+    const reducer = undoable(data, config);
+    const { history, ...newState } = reducer(state, action);
+
+    if (limit === 0) {
+      return { ...newState, past: [{}] };
```

Comment:
why pass an empty object to past?

Reply:
oh I probably should leave a comment there. Because redux-undo takes 0 (limit) as no limit. So if the history length coming from cofig is 0, then I manually overwrite past as an empty array. What's more, I added {} in the array because it is required so in one of the tests, so I figured it's best to keep the same format.

Reply:
👍